moveit_ros/occupancy_map_monitor/src/occupancy_map_monitor.cpp

```diff
@@ -182,7 +182,7 @@ void OccupancyMapMonitor::initialize()
     }
   }
   else
-    ROS_ERROR_NAMED(LOGNAME, "Failed to find 3D sensor plugin parameters for octomap generation");
+    ROS_WARN_NAMED(LOGNAME, "Failed to find 3D sensor plugin parameters for octomap generation");
```

Comment:
Why shouldn't this be an error anymore?

Reply:
I had noticed it being output in many of the `moveit_ros` unit tests, so I am worried it is about a bit more than expected in situations where people have some skeleton of the sensor file from the setup assistant but aren't actually planning to use the perception pipeline.  If this isn't the case at all, I can leave it as an error.

Reply:
It just seemed unrelated, your explanation checks out. Also, the error is not handled in any way, so I'm fine with a warning.